config.yml

```diff
@@ -33,6 +33,9 @@ defaults:
     forceSSL: false
     trustProxy: false
 
+  monitor:
+    project: 'aws-provisioner-dev'
```

Comment:
that's already done in `src/main.js` by checking for `profile === 'test'`, but i suspect what we really want is `profile !== 'production'`